pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,23 +1,33 @@
 { stdenv, fetchFromGitHub, cmake, makeWrapper, qttools
-, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, libgpgerror, glibcLocales, libyubikey, yubikey-personalization, libXi, qtx11extras
+, curl
+, glibcLocales
+, libXi
+, libXtst
+, libargon2
+, libgcrypt
+, libgpgerror
+, libmicrohttpd
+, libyubikey
+, qtbase
+, qtx11extras
+, yubikey-personalization
+, zlib
 , withKeePassHTTP ? true
 }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "keepassxc-${version}";
-  version = "2.2.4";
+  version = "2.3.0";
 
   src = fetchFromGitHub {
     owner = "keepassxreboot";
     repo = "keepassxc";
     rev = "${version}";
-    sha256 = "0q913v2ka6p7jr7c4w9fq8aqh5v6nxqgcv9h7zllk5p0amsf8d80";
+    sha256 = "1zch1qbqgphhp2p2kvjlah8s337162m69yf4y00kcnfb3539ii5f";
   };
 
-  patches = [ ./cmake.patch ./darwin.patch ];
-
   cmakeFlags = [
     "-DWITH_GUI_TESTS=ON"
```

Comment:
You'll want to add `"-DKEEPASSXC_BUILD_TYPE=Release"` to avoid a big prerelease warning in the UI.